tests/models/tvlt/test_feature_extraction_tvlt.py

```diff
@@ -0,0 +1,204 @@
+# coding=utf-8
+# Copyright 2023 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the TVLT feature extraction. """
+
+import itertools
+import os
+import random
+import tempfile
+import unittest
+
+import numpy as np
+import torch
```

Comment:
The current error in the feature extraction tests can be resolved by safely importing torch e.g. 

```
if is_torch_available():
    import torch
```
e.g. [here for Speech T5](https://github.com/huggingface/transformers/blob/d4ba6e1a0e8f662f3deadba25d982c6fb5fb772c/tests/models/speecht5/test_feature_extraction_speecht5.py#L33)